client/jetpack-cloud/sections/settings/advanced-credentials/credentials-form/index.tsx

```diff
@@ -129,41 +190,47 @@ const ServerCredentialsForm: FunctionComponent< Props > = ( { onCredentialsSave,
 			<div className="credentials-form__row">
 				<FormFieldset className="credentials-form__server-address">
 					<div className="credentials-form__support-info">
-						<FormLabel htmlFor="host-address">{ translate( 'Server Address' ) }</FormLabel>
-						{ hostInfo && hostInfo.serverAddress && (
+						<FormLabel htmlFor="host-address">{ translate( 'Server address' ) }</FormLabel>
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **20** times:
`translate( 'Destination Server Address' )` *ES Score: 6*